packages/manager/modules/catalog-price/src/translations/Messages_fr_FR.json

```diff
@@ -1,9 +1,13 @@
 {
   "order_catalog_price_free": "Inclus",
   "order_catalog_price_tax_excl_label": "{{ price }} HT",
+  "order_cdn_catalog_price_tax_excl_label": "[ {{ price }} HT ]",
```

Comment:
Are the brackets mandatory ? Maybe we should keep consistency across all orders/upgrade and either have brackets or not

Reply:
it's define like this into the specs.

Reply:
I saw with Marion, she said we removed them

Reply:
fixed